tests/test_itd_service.rb

```diff
@@ -0,0 +1,291 @@
+                    load_bal_method_end_port:      700,
+                    load_bal_method_proto:         'tcp',
+                    load_bal_method_start_port:    200)
+    assert_equal(128, itd.load_bal_buckets)
+    assert_equal(10, itd.load_bal_mask_pos)
+    assert_equal('ip-l4port', itd.load_bal_method_bundle_hash)
+    assert_equal('dst', itd.load_bal_method_bundle_select)
+    assert_equal(700, itd.load_bal_method_end_port)
+    assert_equal(200, itd.load_bal_method_start_port)
+    assert_equal('tcp', itd.load_bal_method_proto)
+    itd = lb_helper(load_bal_mask_pos: 20)
+    assert_equal(itd.default_load_bal_buckets,
+                 itd.load_bal_buckets)
+    assert_equal(20, itd.load_bal_mask_pos)
+    assert_equal(itd.default_load_bal_method_bundle_hash,
+                 itd.load_bal_method_bundle_hash)
+    assert_equal(itd.default_load_bal_method_bundle_select,
+                 itd.load_bal_method_bundle_select)
+    assert_equal(itd.default_load_bal_method_end_port,
+                 itd.load_bal_method_end_port)
+    assert_equal(itd.default_load_bal_method_start_port,
+                 itd.load_bal_method_start_port)
+    assert_equal(itd.default_load_bal_method_proto,
+                 itd.load_bal_method_proto)
+    itd = lb_helper(load_bal_enable:  true,
+                    load_bal_buckets: 256)
+    assert_equal(256, itd.load_bal_buckets)
+    assert_equal(itd.default_load_bal_mask_pos,
+                 itd.load_bal_mask_pos)
+    assert_equal(itd.default_load_bal_method_bundle_hash,
+                 itd.load_bal_method_bundle_hash)
+    assert_equal(itd.default_load_bal_method_bundle_select,
+                 itd.load_bal_method_bundle_select)
+    assert_equal(itd.default_load_bal_method_end_port,
+                 itd.load_bal_method_end_port)
+    assert_equal(itd.default_load_bal_method_start_port,
+                 itd.load_bal_method_start_port)
+    assert_equal(itd.default_load_bal_method_proto,
+                 itd.load_bal_method_proto)
+    itd = lb_helper(load_bal_enable: false)
+    assert_equal(itd.load_bal_enable,
+                 itd.default_load_bal_enable)
+    assert_equal(itd.load_bal_buckets, itd.default_load_bal_buckets)
+    assert_equal(itd.load_bal_mask_pos, itd.default_load_bal_mask_pos)
+    assert_equal(itd.load_bal_method_bundle_hash,
+                 itd.default_load_bal_method_bundle_hash)
+    assert_equal(itd.load_bal_method_bundle_select,
+                 itd.default_load_bal_method_bundle_select)
+    assert_equal(itd.load_bal_method_end_port,
+                 itd.default_load_bal_method_end_port)
+    assert_equal(itd.load_bal_method_start_port,
+                 itd.default_load_bal_method_start_port)
+    assert_equal(itd.load_bal_method_proto,
+                 itd.default_load_bal_method_proto)
+    itd.destroy
+  end
+
+  def test_nat_destination
+    itd = ItdService.new('new_group')
+    if node.product_id =~ /N9/
```

Comment:
Please use `validate_property_excluded?` instead of hardcoding the platform.

https://github.com/cisco/cisco-network-node-utils/blob/6a0a58a2bf5627b6c91a97a5ae2abb18e2488cb4/tests/test_portchannel_global.rb#L55